pages/dkp/kommander/1.4/release-notes/1.4.2/index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.4.2
+title: Release Notes Kommander 1.4.2
+menuWeight: 20
+excerpt: View release-specific information for Kommander 1.4.2
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Kommander&reg; version 1.4.2 was released on September 23, 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+## Features/Improvements
+
+### Bug fixes
+
+- Fixed an issue where addon links were disabled when the `konvoyconfig-kubeaddons` `ConfigMap` did not exist. Resolves (COPS-6997).
+
+- Because it could not be determined if Grafana was deployed in Federated Clusters, the federated Grafana dashboard ConfigMap was removed. (COPS-6999)
```

Comment:
It's not that it can't be determined, the problem is that federation of Grafana doesn't work so  it's disabled.

```suggestion
- Because Grafana cannot be enabled for Federated Clusters, the federated Grafana dashboard tile was removed. (COPS-6999)
```